### ISSUE TITLE
Adopters Page

### ISSUE THREAD

**Issue by brianking** [EclipseFdn/ecdtools.eclipse.org#42]
Create a page showcasing/listing adopters of our Cloud Dev Tools.

**Comment by brianking** [comment]
The process being used at https://iot.eclipse.org/adopters/ is a good model to use here. It is composed of at least these parts:

1. The page itself : https://github.com/EclipseFdn/iot.eclipse.org/blob/master/content/adopters/_index.md
2. Instructions on how to be listed, as a separate page : https://github.com/EclipseFdn/iot.eclipse.org/blob/master/content/adopters/how-to-be-listed-as-an-adopter/_index.md
3. The YAML file for storing the lisitings : https://github.com/EclipseFdn/iot.eclipse.org/blob/master/data/adopters.yml
4. Logos of adopters in SVG format stored in /static/assets/images/adopters/

**Comment by autumnfound** [comment]
We should add this to the base theme, so that if we need it on any other sites it can be as easy as adding a tag and the data file! 

https://github.com/EclipseFdn/hugo-solstice-theme/issues/148

**Comment by brianking** [comment]
That works for me!

**Comment by chrisguindon** [comment]
> We should add this to the base theme, so that if we need it on any other sites it can be as easy as adding a tag and the data file!
> 
> [EclipseFdn/hugo-solstice-theme#148](https://github.com/EclipseFdn/hugo-solstice-theme/issues/148)

I do not want to duplicate what we did for IoT for ECD Tools but I agree that we will need to include this feature in our base theme.

The plan is to create a new project/repository to allow us to manage adopters for every Eclipse project instead of duplicating this effort for each working group. 

WG sites will then display the projects that they are interested in. The list of projects will come from projects.eclipse.org: https://projects.eclipse.org/api/projects?working_group=cloud-development-tools

**Comment by brianking** [comment]
That sounds like a good plan, thanks for the update. Let me know if/how I can help as you work on this.

**Comment by chrisguindon** [comment]
Sure! 

The ecdtools.eclipse.org website will be the first consumer of this new process! We will need your help to test and confirm that all is well before we start rolling this out to our other WG sites!

**Comment by chrisguindon** [comment]
Update:

We are currently making some progress on this via https://github.com/EclipseFdn/eclipsefdn-project-adopters

**Comment by chrisguindon** [comment]
https://github.com/EclipseFdn/eclipsefdn-project-adopters is now our official home for adopters!

I will create a PR to get this project started. @flora8984461 I will expect you to take it over once the PR is up!

**Comment by chrisguindon** [comment]
We can now get started:
https://deploy-preview-83--eclipsefdn-ecdtools.netlify.app/

https://deploy-preview-83--eclipsefdn-ecdtools.netlify.app/adopters/ is empty seems we now need to find organizations that wished to become adopters of ECD tools project.

We also created an How to be listed page:
https://deploy-preview-83--eclipsefdn-ecdtools.netlify.app/adopters/how-to-be-listed-as-an-adopter/

The content that I am currently using is from the IoT website.

@brianking Can you provide content better suited for ECD Tools for both pages?

**Comment by croundy** [comment]
@chrisguindon @brianking this is a draft content page for the ECD tools web redesign. Please ignore any comments that don't make sense or apply to this particular issue. Please let me know if anything needs clarification. Thanks! https://docs.google.com/document/d/1hD4_gG1K9yKNNBphqsLSuI-qjw2pfSHPD8--T49TE2M/edit

**Comment by chrisguindon** [comment]
Thanks!

Can you share this document with my team? @flora8984461 will be picking up this task from here!

**Comment by flora8984461** [comment]
> @chrisguindon @brianking this is a draft content page for the ECD tools web redesign. Please ignore any comments that don't make sense or apply to this particular issue. Please let me know if anything needs clarification. Thanks! https://docs.google.com/document/d/1hD4_gG1K9yKNNBphqsLSuI-qjw2pfSHPD8--T49TE2M/edit

Hi @croundy Can I have the access to the doc? My email is yi.liu@eclipse-foundation.org  
Thanks!

**Comment by croundy** [comment]
@flora8984461 @chrisguindon I have added you both to the doc. Thank you!

**Comment by brianking** [comment]
I will reach out to some of our members to seed some entries.

Regarding the instructions at https://deploy-preview-83--eclipsefdn-ecdtools.netlify.app/adopters/how-to-be-listed-as-an-adopter/, in general they are fine for now and we can make another pass at content once it is live.

However, I have a few minor change requests:

1. "... you will need to make the following changes to the eclipsefdn-project-adopters' codebase". Can you link to that codebase?
2. Not sure if it is a site-wide issue, but the links on that page are not obvious, e.g. on the word 'issue' in "... creating an issue"
3. On the Adopters page, the link goes directly to creating an issue. It should point to this page with instructions.

**Comment by chrisguindon** [comment]
The list of projects associated with ECD Tools comes from this API:
https://projects.eclipse.org/api/projects?working_group=cloud-development-tools

**Comment by flora8984461** [comment]
I made a draft and the preview is:  https://deploy-preview-84--eclipsefdn-ecdtools.netlify.app/adopters/

https://deploy-preview-84--eclipsefdn-ecdtools.netlify.app/adopters/how-to-be-listed-as-an-adopter/

I wonder for the section (Adopter Quotes), I don't think we want to add this section in the adopters page where all adopters are listed? Do you want me to create a section on homepage? And how many adopters will we display? By dynamic banner, do you prefer a "view more" button to show all the rest or using a carousel? And I think I need some adopters' entries to work on this.

I tried to find some social links, not sure if they're correct, please help me verify, thanks! And I added the icons in "Stay Connected" section.
LinkedIn: https://www.linkedin.com/showcase/ecd-tools/
Twitter: https://twitter.com/ecdtools?lang=en
youtube: https://www.youtube.com/playlist?list=PLy7t4z5SYNaQi-Tg57o1b3iYmxCxHxY0C

What's the link for "contact us"  in `Which ECD Tools Project is right for you? Contact Us`?

**Comment by brianking** [comment]
There are now a few PRs to get some ECD adopters for the listing:
https://github.com/EclipseFdn/eclipsefdn-project-adopters/pull/33
https://github.com/EclipseFdn/eclipsefdn-project-adopters/pull/36
https://github.com/EclipseFdn/eclipsefdn-project-adopters/pull/37

**Comment by chrisguindon** [comment]
@brianking This is great news! Thanks for the update!

**Comment by flora8984461** [comment]
We have one merged adopter listing https://deploy-preview-84--eclipsefdn-ecdtools.netlify.app/adopters/

**Comment by croundy** [comment]
Thank you @flora8984461. Great to see it starting to take shape! To your questions:

> I wonder for the section (Adopter Quotes), I don't think we want to add this section in the adopters page where all adopters are listed? Do you want me to create a section on homepage? And how many adopters will we display? By dynamic banner, do you prefer a "view more" button to show all the rest or using a carousel? And I think I need some adopters' entries to work on this.

This is something to add later as it will require that we pull quotes, pictures, etc. from adopters. For now, let's get the page up and we can iterate once we have those assets.. 
 

> I tried to find some social links, not sure if they're correct, please help me verify, thanks! And I added the icons in "Stay Connected" section.
> LinkedIn: https://www.linkedin.com/showcase/ecd-tools/
> Twitter: https://twitter.com/ecdtools?lang=en
> youtube: https://www.youtube.com/playlist?list=PLy7t4z5SYNaQi-Tg57o1b3iYmxCxHxY0C

Yes, they are all correct. Thank you!
 
> What's the link for "contact us" in `Which ECD Tools Project is right for you? Contact Us`?

With the contact link already lower down on the page, let's eliminate the first "Contact Us" as well as the "Get Listed" link since we have the same CTA in the jumbotron.

With this in mind,  for simplicity, let's consolidate that first section to read: 

**Our open source projects drive innovation across a broad spectrum of industries and on both private and public clouds – enabling organizations of all shapes and sizes to accelerate cloud native development with world-class tools.

Here is a small sampling of adopter organizations that benefit from the Eclipse Cloud Development Tools open source community.**

Perhaps we can increase the font size a bit or something to improve the aesthetics of this text. 

With this change, we can probably also eliminate the "Adopters" header and just go directly to the listings

**Comment by flora8984461** [comment]
@croundy Thank you for the info!
I have increased fontsize to 16px, the preview is updated:  https://deploy-preview-84--eclipsefdn-ecdtools.netlify.app/adopters/

Thanks!

**Comment by chrisguindon** [comment]
Hi All,

I just want to post a note to make sure that we don't lose focus!

The expectation here is that we deliver a new adopters page similar to the one we created for iot.eclipse.org. 

We can make some small adjustments on these pages for ecdtools as long as they are not too hard to implement and don't require significant changes to our new API.

Anything outside of that should not be considered as a deliverable for this project in Q4.

I will be meeting with @brianking and @croundy later this week to discuss further enhancement.

**Comment by flora8984461** [comment]
Then I think my current page is good to go if @croundy gives me +1? 

> Hi All,
> 
> I just want to post a note to make sure that we don't lose focus!
> 
> The expectation here is that we deliver a new adopters page similar to the one we created for iot.eclipse.org.
> 
> We can make some small adjustments on these pages for ecdtools as long as they are not too hard to implement and don't require significant changes to our new API.
> 
> Anything outside of that should not be considered as a deliverable for this project in Q4.
> 
> I will be meeting with @brianking and @croundy later this week to discuss further enhancement.

**Comment by chrisguindon** [comment]
@flora8984461 We definitely need a +1 but I think so!

@brianking @croundy Do you consider that we have any blockers for pushing this page live?

**Comment by croundy** [comment]
@flora8984461 Looks good! 

Would be nice to have a few more adopter before going live, but from my perspective the page looks ready to go.

One comment about how adopters are shown on the page: In addition to a logo, how easy is it to add the name of the product in which it is being used and a short description? Something like this:

**Red Hat OpenShift 4**
OpenShift is a family of containerization software products developed by Red Hat. Its flagship product is the OpenShift Container Platform — an on-premises platform as a service built around Docker containers orchestrated and managed by Kubernetes on a foundation of Red Hat Enterprise Linux.

**Comment by chrisguindon** [comment]
> @flora8984461 Looks good!
> 
> Would be nice to have a few more adopter before going live, but from my perspective the page looks ready to go.

We can wait until tomorrow to make this live.

> One comment about how adopters are shown on the page: In addition to a logo, how easy is it to add the name of the product in which it is being used and a short description? Something like this:
> 
> **Red Hat OpenShift 4**
> OpenShift is a family of containerization software products developed by Red Hat. Its flagship product is the OpenShift Container Platform — an on-premises platform as a service built around Docker containers orchestrated and managed by Kubernetes on a foundation of Red Hat Enterprise Linux.

I would say hard enough. For starters, we would need to change the current layout of the page. This will probably require a designer.

I also think we need to discuss how we want to manage these product descriptions. Is it a product per project or per adopter? Are we expected to display the same project description on IoT and ECDTOOLS if the same project is listed...

I think we are feature creeping here. We should focus on defining the requirement first.

**Comment by croundy** [comment]
@chrisguindon I prefer to use the term "iterate" ;) 

I like it and am good with it as is and we can "feature creep" later.....we are working with a designer and can build the other nice to haves into that process.

Thanks @flora8984461 for the help and the great work! +1

**Comment by chrisguindon** [comment]
> @chrisguindon I prefer to use the term "iterate" ;)

👍 

> 
> I like it and am good with it as is and we can "feature creep" later.....we are working with a designer and can build the other nice to haves into that process.

I agree, I don't think we need to delay this page for those new features.

> Thanks @flora8984461 for the help and the great work! +1

**Comment by brianking** [comment]
FYI I opened recently https://github.com/EclipseFdn/eclipsefdn-project-adopters/issues/38 to start the discussion about adding more information to listings. This should not block release here, it is meant to start the conversation on how we can do it across all WGs.

+1 from me for release.

**Comment by chrisguindon** [comment]
Great! Thanks @brianking
